src/sentry/static/sentry/app/components/search/sources/commandSource.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 
 import {createFuzzySearch} from 'app/utils/createFuzzySearch';
 import {openSudo} from 'app/actionCreators/modal';
-import Feature from 'app/components/feature';
+import Access from 'app/components/acl/access';
```

Comment:
I find it a bit confusing that you import `AccessContainer` as `Access` when the `acl/access` module also defines an `Access` class.

Reply:
The container is simply there for wrapping the org / user context, it's an implementation detail.

Reply:
ok.